### DISCUSSION TITLE
开发了个 CloudflareST 的安卓 APP，欢迎大家尝试

### DISCUSSION THREAD

**Discussion by xianshenglu** [XIU2/CloudflareSpeedTest#320] [分享]
首先感谢大佬的贡献！

开发 App，主要是因为我有部分定制化的需求，且想在安卓上运行。现在写好了，大家也可以试试。下载在这里

https://github.com/xianshenglu/cloudflare-ip-tester-app/releases

主要是我对历史速度做统计，这样长期下来，方便我自己找到相对稳定的节点。因为我经常是找到一个节点，过一阵子又不行了。

有 bug 或者需求的话，在 https://github.com/xianshenglu/cloudflare-ip-tester-app/issues 提，软件功能截图

![image](https://user-images.githubusercontent.com/23273077/165802850-8a1ba1b1-4dc5-4de0-8e01-eea5efdc4b26.png)

**Comment by yeyuchen198** [comment]
> 

有空试试，安装包有点大了30多M，其实在termux里跑测速脚本也很方便的

**Comment by xianshenglu** [comment]
> > 
> 
> 有空试试，安装包有点大了30多M，其实在termux里跑测速脚本也很方便的

我主要要历史统计哈，因为跑出来的节点，第二天可能就又不行了，，，，我想跑的次数多了后，可以沉淀出来一些比较稳定的；

**Comment by xianshenglu** [comment]
> > 
> 
> 有空试试，安装包有点大了30多M，其实在termux里跑测速脚本也很方便的

我改了下打包方式，现在的包有 11M 的
